PlayFabClientSDK/source/PluginContract.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace PlayFab
+{
+    public enum PluginContract
```

Comment:
I'm skeptical about a hard coded list of these.
The plus side is that it's easier and less arbitrary.
The minus side is that the customer can't define totally new categories.

Reply:
PlayFab_Serializer, PlayFab_Transport (they will be strings in the PluginManager map)

---
In reply to: [200496492](https://github.com/PlayFab/CSharpSDK/pull/97#discussion_r200496492) [](ancestors = 200496492)

Reply:
Per offline discussion, it was decided to keep them. This enum is a list of all plugins that are required for compilation of SDK now. Later, we may introduce absolutely custom plugins (not registered or required by PlayFab SDK) that won't use this enum.

---
In reply to: [200507147](https://github.com/PlayFab/CSharpSDK/pull/97#discussion_r200507147) [](ancestors = 200507147,200496492)